lauren_hw.md

```diff
@@ -0,0 +1,31 @@
+## Summer Goals:
+**Learn general skills:**    
+Git - analysis, lab notebook  
+Shell/bash scripting  
+Data analysis - R/Julia  
+
+**Analyze echo metadata:**   
+Echo - analysis of sequencing results
+
+Analysis:  	
+Converting reads into useful information (taxonomic profiles, functional profiles)  
+Taxonomic profiles - what bugs are there  
+Functional profiles - what genes are there  
+Combine in metagenomic sequencing pipeline: connect to Harvard compute grid  
```

Comment:
Probably not Harvard compute grid, but maybe the MIT one Vanja is working on setting up.